### ISSUE TITLE
sensu-extensions-system-profile-wmi wrong configuration instructions

### ISSUE THREAD

**Issue by FlorinAndrei** [sensu-extensions/sensu-extensions-system-profile-wmi#2]
The readme says to do this:

```
{
  "system_profile": {
    "interval": 10,
    "handler": "graphite",
    "add_client_prefix": true,
    "path_prefix": "system",
    "prefix_at_start": true
  }
}
```

It doesn't work. The name of that block should be instead:

```
{
  "wmi_metrics": {
    "interval": 10,
    "handler": "graphite",
    "add_client_prefix": true,
    "path_prefix": "system",
    "prefix_at_start": true
  }
}
```

**Comment by majormoses** [comment]
@FlorinAndrei could you PR the changes and I will be more than happy to review and get the documentation changed.

**Comment by luisdavim** [comment]
Closed via #3. Thanks @FlorinAndrei
